### DISCUSSION TITLE
Agent Survey Results and Assessment Meeting Poll

### DISCUSSION THREAD

**Discussion by Jegelewicz** [ArctosDB/arctos#6891] [Agent Committee]
Agent Committee Members,

Erica has compiled the Agent Survey results and summarized her interviews with select users related to Agents. We would like to set up a meeting of the Agents Committee in December to go over these results and develop strategy for Agents going forward.

Please indicate your availability in this When2Meet - https://www.when2meet.com/?22142176-K8gvF by Friday, November 3rd.

@ArctosDB/agents-committee

**Comment by lin-fred** [comment]
Awesome!

**Reply by ewommack** [reply]
Added times :)
